internal/librarian/command.go

```diff
@@ -208,6 +208,56 @@ func cleanAndCopyLibrary(state *config.LibrarianState, repoDir, libraryID, outpu
 	return copyLibrary(repoDir, outputDir, library)
 }
 
+func copyLibraryFiles(state *config.LibrarianState, dest, libraryID, src string) error {
```

Comment:
What's the difference between `copyLibraryFiles` and `copyLibrary`? Can we combine them?